### ISSUE TITLE
3.0 Action Plan: Gather items for the plan

### ISSUE THREAD

**Issue by joejuzl** [RasaHQ/rasa#8766]
**Description of Problem**:
As part of #8721 we need to collect the actual items for the plan.
This will determine what will be released in 3.0

Creating the plan will be covered by #8763

Some notes from the brainstorm:
- One documentation version for each major version
- New architecture
- Research owned features
- Loading large datasets
- Python 3.9 support
- Maintenance e.g. dropping deprecations
- Research specific maintenance

**Blockers**:
- #8763

**Definition of Done**:
- [x] `Until By` date for 3.0 possibilities communicated to research (see [here](https://rasa-hq.slack.com/archives/C8LEHMTAM/p1622465247301100))
- [x] Items that require a major release are identified
- [x] Items sorted into likely must haves / nice to haves / can wait
- [x] Issues for any necessary further spikes / investigations of  must haves / nice to haves created
- [x] Items added to the plan

**Comment by wochinge** [comment]
1. One engineer does preprocessing
2. Meeting with entire squad to finalize

**Comment by wochinge** [comment]
@TyDunn Updated the action plan. Please review and close issue if everything is alright. One note: How about creating an extra page property for the team which is responsible for a task (in addition to the current `Assign` field)?

**Comment by m-vdb** [comment]
@wochinge I'm not opposed to that, but how useful would that be?

**Comment by TyDunn** [comment]
@wochinge I reviewed. Everything looks good. I added [one item for Slot Mappings and Forms improvements](https://www.notion.so/rasa/3-0-Action-Plan-d88b6710c67a4a48b9e5f910608d7076?p=8a929b6193a6483d8488779493b1f2cf) in case we decided to do it. If we don't, we can drop it. Feel free to close this issue once your conversation with Maxime is complete

**Comment by wochinge** [comment]
>  I'm not opposed to that, but how useful would that be?

Yeah, it might be rather a nice to have. I think it could be useful in case
* the assignee is e.g. on vacation
* to quickly see the distribution of projects on teams (e.g. to see if there is an unbalance)
* to surface potential coordination effort (if a parent project has tasks from different teams)

@m-vdb

**Comment by m-vdb** [comment]
ok sounds good -- let's add it since it's low effort. Done!
